socialmedia.md

```diff
@@ -1,20 +1,118 @@
+* Recommended Use: Have a page for your school (listed as a company) and share performance related updates, such as exam results, achievements and accolades.
 
- 
+#### Instagram
+* Mission: Capture and share the world’s moments.
+* Audience: Those interested in education, teaching and related topics. Parents and families connected to the school.
+* Recommended Use: Have an Instagram page dedicated to sharing the wider life of your school. All photographs. Snapshots of the school day.
+
+#### Snapchat
+* Mission: Life’s more fun when you live in the moment.
+* Audience: Anyone interested in your school specifically – more likely to be parents, community, pupils (dependent on age restrictions).
+* Recommended Use: Have a school account and capture exciting moments as they happen – photo and video.
+
+#### Pinterest
+* Mission: The world’s catalogue of ideas.
+* Audience: Parents and other teachers looking for inspiration.
+* Recommended Use: Have a school account and create a board for each topic or subject. Pin ideas to the board for other teachers to use, or for parents to see as homework/out of school ideas. Invite your teachers to contribute to the board.
+
+#### Youtube
+* Mission: A forum for people to connect, inform, and inspire others across the globe. Videos, trends and channels.
+* Audience: Anyone interested in your content or theme, i.e. education, school, teaching, children.
+* Recommended Use: To share videos that promote your school and learning environment. Christmas videos, shows, assemblies.
+
+
+### Our Social Media Accounts
+
+The school will have a presence on the following social networks: __Facebook, Twitter, LinkedIn, Instagram, Pinterest, Snapchat and Youtube__. Where possible, the account will be listed as a business or company under the school’s name, rather than as a personal or individual. It is important for us, as a school, to maintain a presence on these platforms to discourage others from creating a derogatory account in our name.
+
+The school will actively post on the following social networks: __Facebook, Twitter and Instagram.__
+Employees at the school are permitted to have personal accounts on any social networks they choose, providing they follow the rules outlines in Part 2 of this document and do not attempt to impersonate the school through an unofficial account.
+
+#### Links to Our Accounts
+* Facebook: __Add Here__
+* Twitter: __Add Here__
+* Instagram: __Add Here__
+* LinkedIn: __Add Here__
+* Snapchat: __Add Here__
+* Pinterest: __Add Here__
+* Youtube: __Add Here__
+
+### General Rules and Restrictions
+
+#### Posting on Social Media
+* Content will be posted to official school pages only.
+* There will be no content posted to a private page or closed group.
+* All posts are considered public, regardless of where they are posted, and should be treated as such.
+* The original post must be posted on a school page or account, but can then be shared by others from this place.
```

Comment:
Maybe consider adjusting to something on the lines of 'all school derived content should be posted via official school accounts only, and then reshared/posted/retweeted by teachers/pupils and parents.